dist/pom.xml

```diff
@@ -29,40 +29,150 @@
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-shuffle_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-shims-aggregator_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <groupId>com.nvidia</groupId>
-       <artifactId>rapids-4-spark-udf_${scala.binary.version}</artifactId>
-       <version>${project.version}</version>
-    </dependency>
-    <dependency>
-       <!-- required for conf generation script -->
-       <groupId>org.apache.spark</groupId>
-       <artifactId>spark-sql_${scala.binary.version}</artifactId>
-       <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <!-- required for conf generation script -->
-      <groupId>org.apache.spark</groupId>
-      <artifactId>spark-hive_${scala.binary.version}</artifactId>
-      <scope>provided</scope>
-    </dependency>
-  </dependencies>
+  <profiles>
+    <profile>
+        <id>default</id>
+        <activation>
+            <activeByDefault>true</activeByDefault>
+        </activation>
+        <dependencies>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-shuffle_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-udf_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <groupId>com.nvidia</groupId>
+               <artifactId>rapids-4-spark-shims-aggregator_${scala.binary.version}</artifactId>
+               <version>${project.version}</version>
+            </dependency>
+            <dependency>
+               <!-- required for conf generation script -->
+               <groupId>org.apache.spark</groupId>
+               <artifactId>spark-sql_${scala.binary.version}</artifactId>
+               <scope>provided</scope>
+            </dependency>
+            <dependency>
+              <!-- required for conf generation script -->
+              <groupId>org.apache.spark</groupId>
+              <artifactId>spark-hive_${scala.binary.version}</artifactId>
+              <scope>provided</scope>
+            </dependency>
+        </dependencies>
+    </profile>
+    <profile>
+        <id>with-classifier</id>
```

Comment:
I wonder if we can get down to one thing on the command-line that activates multiple profiles automatically, e.g.: use a property and then have behavior like this based on that property:

- `with-classifier` automatically turns on if the property is defined regardless of the value
- `release301` only turns on if the property is defined and equal to `301`
- `release302` only turns on if the property is defined and equal to `302`
- etc.

Reply:
if you know how to do turns on regardless of value let me know. I attempted something but it didn't work.  Was planning on trying  again, couldn't find docs on it

Reply:
oh you know what I think I see how to now